www/jxcore/mock.js

```diff
@@ -0,0 +1,1059 @@
+'use strict';
```

Comment:
Isn't this file going to blow up when we merge iOS branch into master branch since it has a different thaliMobileNative?